src/tools/perf/perftest_params.c

```diff
@@ -21,11 +21,8 @@ static void print_memory_type_usage(void)
     ucs_memory_type_t it;
 
     ucs_memory_type_for_each(it) {
-        if (ucx_perf_mem_type_allocators[it] != NULL) {
-            printf("                        %s - %s\n",
-                   ucs_memory_type_names[it],
-                   ucs_memory_type_descs[it]);
-        }
+        printf("                        %s - %s\n", ucs_memory_type_names[it],
+               ucs_memory_type_descs[it]);
```

Comment:
I think that `ucx_perftest` will not work with all available memory types unconditionally.

Before the changes `ucx_perftest -h` showed:
if CUDA is not loaded:
```
     -m <send mem type>[,<recv mem type>]
                    memory type of message for sender and receiver (host)
                        host - System memory
```
if CUDA is loaded:
```
     -m <send mem type>[,<recv mem type>]
                    memory type of message for sender and receiver (host)
                        host - System memory
                        cuda - NVIDIA GPU memory
                        cuda-managed - NVIDIA GPU managed/unified memory
```

After the changes `ucx_perftest -h` will unconditionally show:
```
     -m <send mem type>[,<recv mem type>]
                    memory type of message for sender and receiver (host)
                        host - System memory
                        cuda - NVIDIA GPU memory
                        cuda-managed - NVIDIA GPU managed/unified memory
                        rocm - AMD/ROCm GPU memory
                        rocm-managed - AMD/ROCm GPU managed memory
                        rdma - RDMA device memory
```